ragnarok_formats/src/map.rs

```diff
@@ -55,6 +72,79 @@ bitflags::bitflags! {
         const CLIFF = 0b00001000;
     }
 }
+// An internal dfs function needs generics, doesn't know how to put using dfs.
+// TODO: find a easier way to parse QuadTreeData
```

Comment:
These kind of comments are not really helpful, I would prefer if we just removed them.

Reply:
I was thinking that there are no more simple solution to the problem using stack and would probably need to implement using recursive function to generate QuadTree. But this also means that the recursion function will need to derive all the function generics of the parent function and the code would be a lot more messier.

Reply:
That's fine, but the comment doesn't really have anything to do with that